tasks_downstream.py

```diff
@@ -45,6 +46,11 @@
         "build"
     ]["args"]["ODOO_VERSION"]
 )
+DOCKER_COMPOSE_CMD = (
+    "docker compose"
+    if subprocess.getstatusoutput("docker compose")[0] == 0
+    else "docker-compose"
+)
```

Comment:
You can do it simpler:

```suggestion
DOCKER_COMPOSE_CMD = shutil.which("docker-compose") or "docker compose"
```

Reply:
Oh that's nice, thanks!

Quick question: shouldn't we prioritize the newer version if it's installed?

Reply:
Hi @yajo , I have an issue with your suggestion.
If `shutil.which("docker-compose")` finds the path for the given command, then the old version will be used.
We could do the opposite and search if the new version is installed first, but `shutil.which` wont work, as docker compose is a subcommand for docker.

Should we still prioritize the old version?

Reply:
> `shutil.which` wont work, as docker compose is a subcommand for docker

That's the reason why I suggested the old version before. You can do other more complex algorithms if you prefer, but for me this is one enough as it brings compatibility with both CLIs.